pkg/kubelet/images/puller.go

```diff
@@ -58,8 +59,14 @@ func (pip *parallelImagePuller) pullImage(ctx context.Context, spec kubecontaine
 		}
 		startTime := time.Now()
 		imageRef, err := pip.imageService.PullImage(ctx, spec, pullSecrets, podSandboxConfig)
```

Comment:
so bad that the PullImage returns a string instead of a structure. It would be so much easier to just add a size into response of PullImage :-(